contracts/universalSchemes/VestingScheme.sol

```diff
@@ -46,7 +46,7 @@ contract VestingScheme is UniversalScheme, ExecutableInterface {
     }
 
     // A mapping from the organization (Avatar) address to the saved data of the organization:
-    mapping(address=>mapping(bytes32=>Agreement)) public organizationsData;
+    mapping(address=>mapping(bytes32=>Agreement)) public organizationProposals;
```

Comment:
please revert it back to organizationsProposals

Reply:
I'm not sure where this diff is coming from, it is named `organizationProposals` in the code.

Reply:
It should be name organizationsProposals